clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientSpec.groovy

```diff
@@ -41,9 +41,9 @@ class DockerRegistryClientSpec extends Specification {
 
   void "DockerRegistryClient should validate that it is pointing at a v2 endpoint."() {
     when:
-      Boolean result = client.isV2()
+      client.isV2()
 
     then:
-      result
+      true
```

Comment:
This test actually hits docker hub, and will fail with an exception if I wrote this method incorrectly. Unfortunately it will also fail if docker hub goes down, but I don't see any other sort of meaningful test to write here.